lib/events/team_join.js

```diff
@@ -1,4 +1,27 @@
-module.exports = (data, cb) => {
+//const Group = require('../models/group')
+const slack = require('slack')
+const bluebird = require('bluebird')
+const chat = bluebird.promisifyAll(slack.chat)
+
+const cocTxt = process.env.COC_URL ? `<${process.env.COC_URL}|Code of Conduct>` : 'Code of Conduct'
+const welcome_message_text = process.env.WELCOME_MESSAGE_TEXT || `Welcome! Review the ${cocTxt} before participating in the community.`
+const send_welcome_message = process.env.SEND_WELCOME_MESSAGE === 'FALSE' ? false : true
```

Comment:
We need to figure out a way to make this configurable per-Slack, by Admins. I've been considering adding an admin-only `/config` command. How do you feel about this part of your patch being left unused until it can be configured? Different Slacks are going to have completely different needs as far as their welcome messages go, too. >_<